pkg/metricsprovider/signalfx.go

```diff
@@ -64,13 +64,13 @@ func NewSignalFxClient() (watcher.FetcherClient, error) {
 		Transport: tlsConfig}}, nil
 }
 
-func (s signalFxClient) FetchHostMetrics(host string, window *watcher.Window) ([]watcher.Metric, error) {
+func (s signalFxClient) FetchHostMetrics(host string) ([]watcher.Metric, error) {
 	log.Debugf("fetching metrics for host %v", host)
 	var metrics []watcher.Metric
 	hostQuery := signalFxHostFilter + host + signalFxHostNameSuffix
 
 	for _, metric := range []string{cpuUtilizationMetric, memoryUtilizationMetric} {
-		uri, err := buildMetricURL(hostQuery, metric, window)
+		uri, err := buildMetricURL(hostQuery, metric)
```

Comment:
Why you remove the `window` variable?

Reply:
before your email, I think it's useless, it means setting for Prometheus?